### PR TITLE
Move update-api-object-kubectl-patch.md to be under 'Manage Kubernete…

### LINKED ISSUE #21014: Promote content related to resource patching.
**This is a Feature Request**

We have a pretty good doc on various ways to patching an object:

https://kubernetes.io/docs/tasks/run-application/update-api-object-kubectl-patch/

However, the current location of the content is wrong. It is not about an application. It is about more generic ways to change an API object.

**What would you like to be added**

Revise and move the content to object management section so that readers would enjoy playing with the many ways of patching an object.

### PR DESCRIPTION
**Pull request by prasadkatti**
…s Objects' section

Fixes #21014

### CODE REVIEW

File: content/en/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Update API Objects in Place Using kubectl patch
+title: Update API Objects in place using kubectl patch
```
Comment (sftim) [inline content/en/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md:2]: The original title matches the style convention of existing pages (initial capitalization)

Comment (prasadkatti) [reply content/en/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md]: looks like certain words (in, a, an, for, with, the) are not capitalized

**Comment by prasadkatti** [comment]
@sftim - Updated existing pages to link directly to the new URL. So this is ready for another review.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-21041--kubernetes-io-master-staging.netlify.app/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/

**Pull request merged**